WordPress/src/main/java/org/wordpress/android/ui/reader/utils/ReaderEmbedScanner.java

```diff
@@ -0,0 +1,30 @@
+package org.wordpress.android.ui.reader.utils;
+
+import java.util.HashMap;
+import java.util.regex.Pattern;
+
+public class ReaderEmbedScanner {
+    private final String mContent;
+
+    private final HashMap<Pattern, String> mKnownEmbeds = new HashMap<>();
+
+    public ReaderEmbedScanner(String contentOfPost) {
+        mContent = contentOfPost;
+        mKnownEmbeds.put(Pattern.compile("<blockquote[^<>]class=\"instagram-", Pattern.CASE_INSENSITIVE),
```

Comment:
There's likely nothing we can do about this, but hard-coding the URLs here seems fragile.